### DISCUSSION TITLE
How to publish "Fire and Forget" events

### DISCUSSION THREAD

**Discussion by BenDiBen** [BrighterCommand/Brighter#2008] [Q&A]
Hi!

Does Brighter allow you to publish or send an event that is handled asynchronously without having to wait for all of the associated handlers to complete?

In the [HelloAsyncListeners](https://github.com/BrighterCommand/Brighter/blob/17e483b388e7ea6101fe3240a98ee9b9508b24f9/samples/HelloAsyncListeners/RunCommandProcessor.cs#L20) project there is a comment that seems to imply that this should be possible, but I don't understand what the comment is suggesting, since the implementation there does `await` the `PublishAsync` call but it still does not continue to until all of the event handlers are done executing.

Thanks!

**Comment by preardon** [comment]
Hi, 

When you ask about fire and forget events are you talking about over a bus?

Send and Publish are in process
Deposit is for out of process
